language-plutus-core/src/Language/PlutusCore/Type.hs

```diff
@@ -268,7 +268,7 @@ data TermF tyname name a x = VarF a (name a)
                            | UnwrapF a x
                            | WrapF a (tyname a) (Type tyname a) x
                            | ErrorF a (Type tyname a)
-                           deriving (Functor)
+                           deriving (Functor, Traversable, Foldable)
```

Comment:
The `Traversable` instance should be removed.

Reply:
It's useful for writing algebras. You use the one for `TypeF` in exactly the same way :)